prow/config.yaml

```diff
@@ -2629,6 +2629,47 @@ presubmits:
         hostPath:
           path: /mnt/disks/ssd0
 
+  - name: pull-test-infra-bazel-canary
+    agent: kubernetes
+    context: pull-test-infra-bazel-canary
+    branches:
+    - master
+    always_run: false
+    rerun_command: "/test pull-test-infra-bazel-canary"
+    trigger: "(?m)^/test pull-test-infra-bazel-canary,?(\\s+|$)"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/bazelbuild:latest-0.7.0
```

Comment:
do we need to also set `imagePullPolicy: Always`?

Reply:
done.